coderd/agentapi/boundary_logs.go

```diff
@@ -26,6 +26,7 @@ func (a *BoundaryLogsAPI) ReportBoundaryLogs(ctx context.Context, req *agentprot
 		switch r := l.Resource.(type) {
 		case *agentproto.BoundaryLog_HttpRequest_:
 			if r.HttpRequest == nil {
+				a.Log.Warn(ctx, "empty http request resource")
```

Comment:
Warning logs have to be diagnostically useful; currently this will leave operators with a dead end. Can you include some fields in here to help triage the issue?

Reply:
The logger `a.Log` already has fields attached to it (e.g. workspace name, owner, request ID, agent name), so I only added the workspace ID here and a few lines down.